src/main/java/org/snomed/snowstorm/core/data/domain/Axiom.java

```diff
@@ -119,12 +120,49 @@ public Axiom setRelationships(Set<Relationship> relationships) {
 		return this;
 	}
 
-	@JsonIgnore
 	public ReferenceSetMember getReferenceSetMember() {
 		return referenceSetMember;
 	}
```

Comment:
I've removed this annotation because I needed the Axiom's ReferenceSetMember whilst constructing the middle column.